rcp/org.testeditor.rcp4.views.tcltestrun/src/org/testeditor/rcp4/views/tcltestrun/model/TestCaseExecution.xtend

```diff
@@ -0,0 +1,22 @@
+/*******************************************************************************
+ * Copyright (c) 2012 - 2016 Signal Iduna Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ * 
+ * Contributors:
+ * Signal Iduna Corporation - initial API and implementation
+ * akquinet AG
+ * itemis AG
+ *******************************************************************************/
+package org.testeditor.rcp4.views.tcltestrun.model
+
+import org.eclipse.xtend.lib.annotations.Data
+
+@Data
+class TestCaseExecution {
```

Comment:
I created this one to store the selection of the current test run. I plan to use it to show the test cases and the steps, which have to be hit by the test run.
Currently it is more a leaf in the code for further work.